### DISCUSSION TITLE
how to scroll back when ssh to a remote host via terminal

### DISCUSSION THREAD

**Discussion by jiazhan** [Eugeny/tabby#5451] [Q&A] [answered]
Hi team : 
I have a question as subject:
How to reproduce : 
version 1.0.169 on intel chip macos bigsur 11.6
1. open a terminal and run `ssh a_demo_remotehost`
2. when I try to scroll up with mouse when logged in the remotehost, it will only display the previous command but not scroll. I did not find a place to set it.
can you help to review ?

**Comment by jiazhan** [comment, marked as answer]
Oh, I think I am wrong. I can scroll back now. sorry for confuse here. please close it .

**Reply by jiazhan** [reply]
it turned out that I have to use screen copy mode or configure screen configuration files as 
https://www.shellhacks.com/screen-scroll-up-scrollback-buffer/
but it works fine with iterm2 without such configurations

**Reply by rahulsinghk** [reply]
could you please explain? 
i am facing the same issue

**Comment by lawweiliang** [comment]
@jiazhan  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
